apitest.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
Likewise with last comment; If this is just a test, it should be removed from the project.